scripts/us_epa/ghgrp/import_data/schema.mcf

```diff
@@ -0,0 +1,55 @@
+Node: dcs:GreenhouseGas
+typeOf: schema:Class
+subClassOf: dcs:Enumeration
+description: "Gases that trap heat in the atmosphere."
+descriptionUrl: "https://www.epa.gov/ghgemissions/overview-greenhouse-gases"
+
+Node: dcs:EmissionSourceEnum
+typeOf: schema:Class
```

Comment:
Prod may still complain about requiring names for the schema things... I think.

Reply:
i'll add names here and elsewhere just to be on the safe side